src/content/peerconnection/dtmf/js/test.js

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (c) 2016 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+ /* eslint-env node */
+
+'use strict';
+// This is a basic test file for use with testling.
+// The test script language comes from tape.
+var test = require('tape');
+
+var webdriver = require('selenium-webdriver');
+var seleniumHelpers = require('webrtc-utilities').seleniumLib;
+
+test('DTMF tones', function(t) {
+  if (process.env.BROWSER === 'firefox') {
+    t.pass('Firefox not supported yet');
+    t.end();
+    return;
+  }
+  var driver = seleniumHelpers.buildDriver();
+
+  driver.get('file://' + process.cwd() +
+      '/src/content/peerconnection/dtmf/index.html')
+  .then(function() {
+    t.pass('page loaded');
+    return driver.findElement(webdriver.By.id('callButton')).click();
+  })
+  .then(function() {
+    return driver.wait(function() {
+      return driver.executeScript(
+          'return document.querySelector(\'#dtmfStatus\').innerHTML === ' +
+          '\'DTMF activated\';');
+    }, 30 * 1000);
+  })
+  .then(function() {
+    t.pass('DTMF activated');
+    // Set the tones to be sent.
+    return driver.executeScript(
+        'document.querySelector(\'#tones\').value = \'1 # , 9 \'');
```

Comment:
That's what I get when sending the tones, I'm just matching the output like I wrote [here](https://github.com/webrtc/samples/pull/767/files/5075286359ee1a59bfdec47473387e007ce0ee44#diff-821d966122db351688988c1a9a2358acR51)